UI/win-update/updater/updater.cpp

```diff
@@ -1873,6 +1873,18 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE, LPWSTR lpCmdLine, int)
 
 	is32bit = wcsstr(cwd, L"bin\\32bit") != nullptr;
 
+	if (!IsWindows10OrGreater()) {
+		MessageBox(
+			nullptr,
+			L"OBS Studio 28 and newer no longer support Windows 7,"
+			L" Windows 8, or Windows 8.1. You can disable the"
+			L" following setting to opt out of future updates:"
+			L" Settings -> General -> General -> Automatically"
```

Comment:
I would recommend using Unicode arrows here to match how OBS itself presents these guides.

Reply:
Could you recommend a specific character?

Reply:
We use → in the existing texts.

Reply:
Pushed changes.